src/Inciter/Scheme.h

```diff
@@ -218,7 +218,7 @@ class Scheme : public SchemeBase {
     //!   using the last argument as default.
     template< typename Op, typename... Args, typename std::enable_if<
       std::is_same< Op, tag::elem >::value, int >::type = 0,
-      typename std::enable_if< sizeof...(Args) == 11, int >::type = 0 >
+      typename std::enable_if< sizeof...(Args) == 8, int >::type = 0 >
```

Comment:
Just a note: all of this will go away once https://github.com/quinoacomputing/quinoa/pull/194 is merged.